scripts/git-release.ts

```diff
@@ -0,0 +1,52 @@
+import { execSync } from "child_process";
+import fs from "fs";
+import simpleGit from "simple-git";
+
+const git = simpleGit();
+
+main();
+
+async function main() {
+  const changelogFile = fs.readFileSync("../CHANGELOG.md", "utf-8");
+
+  const newestEntry = changelogFile.split("\n## ");
+  newestEntry.shift(); // Remove the first item, it's the main heading.
+
+  const releaseLines = newestEntry[0].trim().split("\n");
```

Comment:
The name `newestEntry` is a bit misleading since it contains all entries. Suggestion:
```suggestion
  const newestEntry = changelogFile.split("\n## ")[1]; // The first item is the main heading.
  const releaseLines = newestEntry.trim().split("\n");
```